pkg/v1/tkg/test/tkgctl/shared/ceip.go

```diff
@@ -118,3 +133,83 @@ func verifyTelemetryJobURL(context context.Context, url string, mcProxy *framewo
 
 	return errors.New("URL not found in the telemetry cron job")
 }
+
+func verifyTelemetryJobRunning(context context.Context, mcProxy *framework.ClusterProxy) error {
+	var (
+		err          error
+		selectors    = []crtclient.ListOption{crtclient.InNamespace(telemetryNamespace)}
+		client       = mcProxy.GetClient()
+		pollInterval = 30 * time.Second
+		pollTimeout  = 90 * time.Second
+	)
+
+	scheme := mcProxy.GetScheme()
+	batchv1.AddToScheme(scheme)
+
+	cronJob := &v1beta1.CronJob{}
+	if err = client.Get(context, types.NamespacedName{Name: telemetryName, Namespace: telemetryNamespace}, cronJob); err != nil {
+		return err
+	}
+	// updating the telemetry cron job schedule to "* * * * *" so that the cronjob can be scheduled to run within the next 59 seconds
+	cronJob.Spec.Schedule = "* * * * *"
```

Comment:
Suggestion: please deepCopy and make changes to the copied object instead. It is risky to change the pointer directly. The same to L205 as well.

Reply:
Thanks for this. Will fix